vars/deployOpenShiftSnapshot.groovy

```diff
@@ -60,8 +60,8 @@ def call(body) {
             export FABRIC8_SSO_API_URL="https://sso.prod-preview.openshift.io/"
             export FABRIC8_AUTH_API_URL="https://auth.prod-preview.openshift.io/api/"
 
-            export OPENSHIFT_CONSOLE_URL="https://api.free-stg.openshift.com/console/"
```

Comment:
The env var looks like this change makes sense but if it has been working do we need this change?

Either it isn't used OR it already works OR the change in to the proxy requires this for some reason.

Reply:
That value is incorrect now - it does not work.  Setting them all consistently will help with removal in our next PR.